flink-end-to-end-tests/test-scripts/test_pyflink.sh

```diff
@@ -223,13 +223,23 @@ function read_msg_from_kafka {
 }
 
 function cat_jm_logs {
-     local log_file_name=${3:-standalonesession}
-     cat $FLINK_LOG_DIR/*$log_file_name*.log
+  cat_logs_in_order "standalonesession"
 }
 
 function cat_tm_logs {
-	local logfile="$FLINK_LOG_DIR/flink*taskexecutor*log"
-	cat ${logfile}
+  cat_logs_in_order "flink*taskexecutor"
+}
+
+function cat_logs_in_order {
+  local log_file_name_pattern log_file_name_base ordered_file_names
+  log_file_name_pattern="$1"
+  log_file_name_base="*${log_file_name_pattern}*.log"
+
+  # this prints the logs as if all logs are stored in a single file,
+  # i.e. ensuring the right order of log lines based on the rollover
+  # strategy used in Flink by default
+  ordered_file_names=$(ls $log_file_name_base.* | sort)
+  cat ${ordered_file_names} $log_file_name_base
```

Comment:
why is it that pyflink has this special function?

Reply:
man let's just remove this stuff and have the logs be uploaded like _in every other test_.

Reply:
good point. I added it as a separate commit...